helm/charts/hpe-csi-driver/README.md

```diff
@@ -69,9 +69,34 @@ helm install my-hpe-csi-driver hpe-storage/hpe-csi-driver -n hpe-storage -f myva
 ### Upgrading the chart
 
 Due to the [helm limitation](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations) to not support upgrade of CRDs between different chart versions, helm chart upgrade is not supported.
-
 Our recommendation is to uninstall the existing chart and install the chart with the desired version. CRDs will be preserved between uninstall and install.
 
+## Upgrading 2.0.0 to 2.1.0
+Before version 2.0.0 is uninstalled, the following CRDs needs to be updated.
+```
+kubectl apply -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/helm/charts/hpe-csi-driver/crds/hpevolumeinfos_v2_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/helm/charts/hpe-csi-driver/crds/hpevolumegroupinfos_v2_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/helm/charts/hpe-csi-driver/crds/snapshotgroupinfos_v2_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/helm/charts/hpe-csi-driver/crds/hpereplicated_deviceinfo_v2_crd.yaml
+```
+If there are HPE Alletra 9000, Primera or 3PAR Remote Copy Groups configured on the cluster, follow the below steps before uninstallation
+```
+Create the job using the below commands, which will modify the rcg-info record to the new key RCGCreatedByCSP.
+curl -O  https://raw.githubusercontent.com/hpe-storage/co-deployments/master/yaml/rcg-info/v1.0.0/convert-rcg-info.yaml
+Note: If namespace is other than hpe-storage modify the namespace parameter in convert-rcg-info.yaml file.
+kubectl create -f convert-rcg-info.yaml
```

Comment:
Rewrite this into: 
Change `kubectl` context into the `Namespace` where the HPE CSI Driver is installed. The most common is "hpe-storage".

```
kubectl config set-context --current --namespace=hpe-storage
```

Create the `Job` using the below commands, which will modify the "rcg-info" record to the new key "RCGCreatedByCSP".

```
kubectl apply -f https://raw.githubusercontent.com/hpe-storage/co-deployments/master/yaml/rcg-info/v1.0.0/convert-rcg-info.yaml
```